service/request.go

```diff
@@ -27,6 +29,18 @@ func NewHTTPStatusError(status int, err error) *HTTPStatusError {
 	return &HTTPStatusError{Status: status, Err: err}
 }
 
+// IsNotFound returns true if the error is `sql.ErrNoRows` or `os.ErrNotExist`.
+func IsNotFound(err error) bool {
+	switch {
+	case errors.Is(err, sql.ErrNoRows):
+		return true
+	case errors.Is(err, os.ErrNotExist):
```

Comment:
This case is included sake of completeness since nanomdm includes a `FileStorage` implementation in addition to MySQL and PostgreSQL.